### PR TITLE
Upgrade to amber v0.6.1

### PR DESCRIPTION
**Pull request by marksiemers**

### CODE REVIEW

**Comment by marksiemers** [comment]
This is a WIP, I have the site running locally with development settings. I tried running in production mode, but it doesn't seem to use all the settings.

It brings up 4 processes, but doesn't use port 8080, and the logs show everything.

**Comment by eliasjpr** [comment]
@marksiemers since you're at I believe is safe to remove this file https://github.com/amberframework/amberframework.org/blob/master/src/helpers/guide_helper.cr

We only need to update the sidebar when adding/updating the guides navigation.

**Comment by marksiemers** [comment]
@elorest - The `secret_key_base` is required, right? For this site, it probably doesn't matter that it is secret since nothing in the session really needs to remain private, but I think it would just demonstrate bad practice.

**Comment by elorest** [comment]
Yes it is necessary as sessions are based off of it.

**Comment by eliasjpr** [comment]
Do we use session on the site? I don't think we do @elorest

**Comment by marksiemers** [comment]
This looks to be working now, so we should be able to use either PR. This or #93

File: config/environments/production.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+logging:
+  severity: info
+  colorize: false
+  filter:
+    - password
+    - confirm_password
+  skip:
+    -
+  context:
+    - request
```
Comment (marksiemers) [inline config/environments/production.yml]: Should this be removed from production logging?

**Comment by marksiemers** [comment]
@amberframework/core-team - One more approval?

**Pull request merged**